src/popup/router/pages/SuccessTip.vue

```diff
@@ -70,10 +70,11 @@ export default {
     },
   },
   async created() {
-    const { addresses, tab } = await this.$store.dispatch('getWebPageAddresses');
-    console.log(addresses);
-    if (addresses.length) {
-      await axios.post(`${TIP_SERVICE}`, { url: tab.url, address: addresses[0] });
+    if (process.env.IS_EXTENSION) {
+      const { addresses, tab } = await this.$store.dispatch('getWebPageAddresses');
```

Comment:
If there is no check here error will be thrown on mobile, because there is no content script on mobile. Not sure what has to happen on mobile? cc @mradkov

Reply:
On mobile we should still invoke the payfortx when sending tip.